packages/design-system/src/scalars/lib/index.ts

```diff
@@ -1,4 +1,5 @@
-export * from "./currency-list.js";
+/* eslint-disable prettier/prettier */
+export * from "./decorators.js";
```

Comment:
This barrel file should only export files meant to be used on web apps.
The decorators file has storybook code so it should not be part of the final bundle.

Reply:
Removed from the exports!